index.js

```diff
@@ -323,7 +323,7 @@ program
 
       // Create a local HTTP server that reads the OAuth token
       var app = connect();
-      app.use(function(req, res) {
+      app.use(function (req, res) {
```

Comment:
Don't change this next time. (We should use an arrow function if we change).

Reply:
Thanks. The space after function is actually added by Eslint. I will pay more attention next time.